controls/SV-204459.rb

```diff
@@ -8,41 +8,31 @@
     exist, patches are usually made available by the vendor to resolve the problems. If the most recent security patches
     and updates are not installed, unauthorized users may take advantage of weaknesses in the unpatched software. The
     lack of prompt attention to patching could result in a system compromise.'
-  desc 'check', 'Verify the operating system security patches and updates are installed and
-      up to date. Updates are required to be applied with a frequency determined by
-      the site or Program Management Office (PMO).
+  desc 'check', 'Verify the operating system security patches and updates are installed and up to date. Updates are required to be applied with a frequency determined by the site or Program Management Office (PMO). 
 
-      Obtain the list of available package security updates from Red Hat. The URL
-      for updates is https://rhn.redhat.com/errata/. It is important to note that
-      updates provided by Red Hat may not be present on the system if the underlying
-      packages are not installed.
+Obtain the list of available package security updates from Red Hat. The URL for updates is https://rhn.redhat.com/errata/. It is important to note that updates provided by Red Hat may not be present on the system if the underlying packages are not installed.
 
-      Check that the available package security updates have been installed on
-      the system with the following command:
+Check that the available package security updates have been installed on the system with the following command:
 
-          # yum history list | more
-          Loaded plugins: langpacks, product-id, subscription-manager
-          ID     | Command line             | Date and time    | Action(s)      |
-      Altered
+# yum history list | more
+Loaded plugins: langpacks, product-id, subscription-manager
+ID     | Command line             | Date and time    | Action(s)      | Altered
```

Comment:
This was not the table contents that showed up when testing on a local instance.

Actual Results:
```
inspec> command('yum history list').stdout.split("\n")
=> ["Loaded plugins: langpacks, product-id, search-disabled-repos",
 "ID     | Login user               | Date and time    | Action(s)      | Altered",
 "-------------------------------------------------------------------------------",
 "     8 |  <vagrant>               | 2016-12-21 00:37 | Install        |    6  <",
 "     7 |  <vagrant>               | 2016-12-20 22:20 | Install        |   16 > ",
 "     6 |  <vagrant>               | 2016-12-20 22:19 | Install        |    3   ",
 "     5 |  <vagrant>               | 2016-12-20 22:17 | Install        |    4   ",
 "     4 |  <vagrant>               | 2016-12-20 19:36 | Erase          |    2   ",
 "     3 |  <vagrant>               | 2016-12-20 19:34 | Install        |   30   ",
 "     2 |  <vagrant>               | 2016-12-20 19:29 | I, U           |  306 EE",
 "     1 | System <unset>           | 2016-04-20 19:30 | Install        |  507   ",
 "history list"]
 ```